src/util/LoadingButton.tsx

```diff
@@ -12,15 +13,42 @@ interface Props {
   onClick?: onClick;
 }
 
+function renderButton(props: Props) {
+  if (props.disabled) {
+    return (<>
+      <ReactTooltip place='left' type='dark' effect='solid'/>
+      <a data-tip='This lab is currently disabled'>
```

Comment:
This tooltip needs to be passed in. We are/will be using the loading button in more than one places. Call the prop, disabledToolTip